rackcity/views/asset_views.py

```diff
@@ -234,6 +237,54 @@ def save_mac_addresses(asset_data, asset_id):
         raise MacAddressException(failure_message)
 
 
+def save_power_connections(asset_data, asset_id):
```

Comment:
I feel like this function and the mac address equivalent could live somewhere better, but I'm also not sure we have a better place currently. Might be part of a larger discussion